### DISCUSSION TITLE
Issues about dynamic parameter configuration of IEEE 68-bus system

### DISCUSSION THREAD

**Discussion by Albizzian** [CURENT/andes#538] [Q&A] [answered]
Hello, I downloaded the IEEE 68-bus system dynamic data from Internet with PSS/E format. I converted it to ANDES format and added governor, exciter and PSS models manually. But it seems that the generator' rotor oscillate after a power disturbance. Could you help me to configure the paraments to have a good performance on doing the frequency response simulation? Documents are attached below.Thank you very much! 
![image](https://github.com/CURENT/andes/assets/78994912/de05b6a2-4fd1-4274-93bf-82f1adc5c6ae)
[ieee68.xlsx](https://github.com/CURENT/andes/files/15145005/ieee68.xlsx)
[ieee_68_PSS.zip](https://github.com/CURENT/andes/files/15145034/ieee_68_PSS.zip)

**Comment by cuihantao** [comment, marked as answer]
Try getting rid of the TG deadbands (set to zero). Also try setting `Tw` of `BusROCOF` and `BusFreq` to 0.1.

**Reply by Albizzian** [reply]
Hantao,Thanks for your suggestion and I tried it, but it seems doesn't effective.  I found when I adjust the IEEEST model's paraments, the generator' rotors have different curves and the oscillation decrease or increase as below. Maybe the key is the paraments of PSS model? But I don't know how to properly adjust it.
![image](https://github.com/CURENT/andes/assets/78994912/45184440-ab24-4999-b103-c7d3739a49e3)
![image](https://github.com/CURENT/andes/assets/78994912/02578c70-cf47-47e2-8285-aa81ed9d67b2)

**Comment by cuihantao** [comment]
Maybe take all the PSSes out and see if it fixes it?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Albizzian** [reply]
I removed the PSSes, but it still doesn't work and stay the same. I am confused about which model will influence the oscillation now. 
![image](https://github.com/CURENT/andes/assets/78994912/d396bbbf-4996-4c1f-a187-f3d773f34b4a)

**Comment by cuihantao** [comment]
You can run the eigenvalue analysis and check the participation factor of states.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by Albizzian** [reply]
Thanks! Sounds a good way. I'll try it.
